README.md

```diff
@@ -35,7 +35,7 @@ You also need to create a JOSN configuration file:
 ```
 
 The files have the following meaning:
-* case - The DIVD case number
+* case - The DIVD case number, this needs to be an existing folder in Google Drive
```

Comment:
Je kunt alleen niet zien welke folders er bestaan op de case drive of niet. Ik zou zeggen dat er een folder voor deze case moet zijn op de case drive waar je toegang toe hebt.